src/app/movies/movies.component.ts

```diff
@@ -21,6 +22,8 @@ export class MoviesComponent {
   }
 
   parseUserInput(movieInput: string): Movie {
+    const getVideoMetadata = getVideoId.getVideoId(movieInput);
```

Comment:
a zmienna nie powinna się nazywać getVideoMetadata, get to czasownik - czasownikami nazywamy funkcje. Czyli kiedy funkcja to getCośtam, to jej wynik można zapisać w zmiennej cośtam ;)